### PR TITLE
[Jetchat] Address and remove TODOs

### PR DESCRIPTION
**Pull request by JoseAlcerreca**

### CODE REVIEW

File: Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt [hunk 1]
```diff
@@ -59,8 +60,8 @@ class NavActivity : AppCompatActivity() {
                 ) {
                     val scaffoldState = rememberScaffoldState()
 
-                    val openDrawerEvent = viewModel.drawerShouldBeOpened.observeAsState()
-                    if (openDrawerEvent.value == true) {
+                    val openDrawerEvent = viewModel.drawerShouldBeOpened.collectAsState()
```
Comment (nickbutcher) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt:63]:
```suggestion
                    val drawerOpen by viewModel.drawerShouldBeOpened.collectAsState()
```

File: Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt [hunk 2]
```diff
@@ -277,18 +282,14 @@ fun Messages(
...
     isFirstMessageByAuthor: Boolean,
     isLastMessageByAuthor: Boolean
 ) {
-    // TODO: get image from msg.author
-    val painter = if (isUserMe) {
-        painterResource(id = R.drawable.ali)
-    } else {
-        painterResource(id = R.drawable.someone_else)
-    }
+    val painter = painterResource(id = msg.authorImage)
```
Comment (nickbutcher) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/Conversation.kt:291]: Inline?

**Review by nickbutcher — APPROVED** [review]
A couple of small suggestions, otherwise LGTM.

File: Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt (hunk 1 above)
Comment (JoseAlcerreca) [reply Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt:63]: There was a reason why I didn't do `by` :shrug:

Comment (JoseAlcerreca) [reply Jetchat/app/src/main/java/com/example/compose/jetchat/NavActivity.kt:63]: Not committing this because it's not complete

**Pull request merged**